en/mkdocs.yml

```diff
@@ -79,7 +79,11 @@ nav:
         - Create API Revisions: design/create-api/create-api-revisions.md
         - Add Custom Properties to APIs: design/create-api/adding-custom-properties-to-apis.md
         - Change the Thumbnail of an API: design/create-api/change-api-thumbnail.md
-        - Create Prototype APIs: design/prototype-api/create-a-prototype-api.md
+        - Create Prototype APIs: 
+            - Overview: design/prototype-api/overview.md
+            - Mock Implementation with API Gateway: design/prototype-api/create-mocked-js-api.md
+            - Mock Implementation with Choreo Connect: design/prototype-api/create-mocked-oas-api.md
```

Comment:
Instead of using `prototyped` APIs shall we use the term `Mock API` ?

Reply:
Mock API is only one type of Prototype APIs. We can have an Prototype API without an Mock Impl and rather with an actual backend URL too.